docs/examples/cookbook/drag-and-drop.md

```diff
@@ -284,7 +284,7 @@ local allEntries = scope:ForValues(
 		local itemPosition = scope:Value(nil)
 		return scope:TodoEntry {
 			Item = item,
-			Parent = scope:Computed(function(use)
+			Parent = scope:Computed(function(use, scope)
```

Comment:
Scope shouldn't be provided as it is never used
```suggestion
			Parent = scope:Computed(function(use)
```

Reply:
reverted